make-distribution.sh

```diff
@@ -57,7 +57,7 @@ while (( "$#" )); do
     --hadoop)
       echo "Error: '--hadoop' is no longer supported:"
       echo "Error: use Maven profiles and options -Dhadoop.version and -Dyarn.version instead."
-      echo "Error: Related profiles include hadoop-0.23, hdaoop-2.2, hadoop-2.3 and hadoop-2.4."
+      echo "Error: Related profiles include hadoop-0.23, hadoop-1, hadoop-2.2, hadoop-2.3 and hadoop-2.4."
```

Comment:
Good catch on the spelling; I think we don't need to mention `hadoop-2.2` now as its a no-op.